### PR TITLE
Replace .png files with xcassets approach, drop pre ios 7 support

### PR DESCRIPTION
**Pull request by jvandijk**
This PR resolves the issues that existed with non loading assets. I've removed the NSBundle approach in favor of the advised xcassets approach. xcassets support non-retina, retina and iPhone-6 plus formats by default.

Important change in the usage is mentioned in the README. One should still drop the xcassets in the resources folder, but not copy the contents.

Simple explanation of using xcassets : http://guides.codepath.com/ios/Adding-Image-Assets

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ npm install --save react-native-nmrangeslider-ios
 Follow the [procedure to link the library in XCode](https://facebook.github.io/react-native/docs/linking-libraries-ios.html#manual-linking) 
 or run `rnpm link react-native-nmrangeslider-ios` using [rnpm](https://github.com/rnpm/rnpm).
 
-
-Then drag and drop the "DefaultTheme7" folder from `node_modules/react-native-nmrangeslider-ios/` to your
-XCode Resources folder. Select "copy files if necessary" and link them to your project. 
+Open your project in XCode if necessary and go into `Libraries > RNNMRangeSlider.xcodeproj > NMRangeSlider` and drag
+and drop the "Slider.xcassets" to the Resources folder. In the dialog, do NOT tick "copy files if necessary" because
+the files are already part of this project.
```
Comment (adri) [inline README.md:25]: Should we mention how to change the images?

File: README.md [hunk 2]
```diff
@@ -32,13 +32,23 @@ import NMRangeSliderIOS from 'react-native-nmrangeslider-ios';
...
+  trackColor={#00ff00}
+  onChange={onChangeFunction}
+  disabled={false}
   style={{ width: 300, height: 50 }}
 />
 ```
 
+# Changelog
+
+- 1.3.0 Changed image assets into xcassets structure
```
Comment (adri) [inline README.md:48]: Awesome changelog!

**Comment by adri** [comment]
This is great!

File: README.md (hunk 1 above)
Comment (jvandijk) [inline README.md:25]: There is more that needed, if images are smaller also code tweaks are necessary. So let's leave that for now.

**Pull request merged**